relbench/data/task_node.py

```diff
@@ -93,3 +107,34 @@ def __init__(self, dataset: str, process: bool = False) -> None:
 
         def pack_tables(self, root: Union[str, os.PathLike]) -> Tuple[str, str]:
             return _pack_tables(self, root)
+
+    def stats(
+        self, split: Literal["train", "val", "test"] = "train"
+    ) -> dict[str, dict[str, Any]]:
+        if split == "train":
+            table = self.train_table
+        elif split == "val":
+            table = self.val_table
+        else:
+            table = self.test_table
+        timestamps = table.df[self.time_col].unique()
+        target_stype = stype.categorical
+        if self.task_type == TaskType.REGRESSION:
+            target_stype = stype.numerical
+        if self.task_type == TaskType.MULTILABEL_CLASSIFICATION:
+            raise ValueError(f"Unsupported task type{self.task_type}")
+        res = {}
+        for timestamp in timestamps:
+            temp_df = table.df[table.df[self.time_col] == timestamp]
+            entity_col_stats = compute_col_stats(
+                temp_df[self.entity_col], stype.categorical
+            )
+            target_col_stats = compute_col_stats(
+                temp_df[self.target_col],
+                target_stype,
+            )
+            res[str(timestamp)] = {
+                self.entity_col: entity_col_stats,
+                self.target_col: target_col_stats,
+            }
+        return res
```

Comment:
what is it used for?

Reply:
This is the train / val / test table statistics, proposed by @rusty1s

Reply:
yes, but where are we using it?

Reply:
Seems to add this for the train table stats listed in the paper

Reply:
Discussed with @rusty1s , we are going to add the target distribution etc. to the appendix

Reply:
Ok!